grafana/Dockerfile

```diff
@@ -17,11 +68,10 @@ ENV PATH=/usr/share/grafana/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bi
 
 WORKDIR $GF_PATHS_HOME
 
-RUN apk --no-cache upgrade && \
```

Comment:
We should keep the apk upgrade call for the final image to assure having latest alpine dependencies installed

Reply:
fixed.